### PR TITLE
Rename go pkg to github.com/istio-ecosystem/sail-operator

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -21,17 +21,17 @@ import (
 	"os"
 	"strings"
 
+	maistraiov1 "github.com/istio-ecosystem/sail-operator/api/v1alpha1"
```
Comment (dgn) [inline cmd/main.go]: let's change the `maistra` here as well :)

File: tests/integration/api/istio_test.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ var _ = Describe("Istio resource", Ordered, func() {
 		workloadNamespace = "istio-test-workloads"
 
 		gracePeriod = 30 * time.Second
-		pilotImage  = "maistra.io/test:latest"
+		pilotImage  = "operator.io/test:latest"
```
Comment (dgn) [inline tests/integration/api/istio_test.go]: nit: maybe sail-operator.io - or just `sail-operator/test:latest`

File: tests/integration/api/istiorevision_test.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ var _ = Describe("IstioRevision resource", Ordered, func() {
 		revName        = "test-istiorevision"
 		istioNamespace = "istiorevision-test"
 
-		pilotImage = "maistra.io/test:latest"
+		pilotImage = "operator.io/test:latest"
```
Comment (dgn) [inline tests/integration/api/istiorevision_test.go]: same here

**Review by dgn — APPROVED** [review]
lgtm, just two nitpicks

File: cmd/main.go (hunk 1 above)
Comment (luksa) [reply cmd/main.go]: I typically omit "maistra" when importing our API. I just use `v1alpha1`. For external APIs, I use the API group name and version (e.g. `corev1`, `appsv1`, ...). Can we agree to use this system everywhere?

File: controllers/istiorevision/istiorevision_controller_test.go [hunk 4]
```diff
@@ -19,12 +19,12 @@ import (
 	"strings"
 	"testing"
 
+	v1 "github.com/istio-ecosystem/sail-operator/api/v1alpha1"
```
Comment (luksa) [inline controllers/istiorevision/istiorevision_controller_test.go]:
```suggestion
	"github.com/istio-ecosystem/sail-operator/api/v1alpha1"
```

File: pkg/kube/finalizers_test.go [hunk 5]
```diff
@@ -19,14 +19,14 @@ import (
 	"os"
 	"testing"
 
+	v1 "github.com/istio-ecosystem/sail-operator/api/v1alpha1"
```
Comment (luksa) [inline pkg/kube/finalizers_test.go]:
```suggestion
	"github.com/istio-ecosystem/sail-operator/api/v1alpha1"
```

File: cmd/main.go (hunk 1 above)
Comment (dgn) [reply cmd/main.go]: sgtm

**Pull request merged**